### PR TITLE
VOTE-951: secondary footer refinements

### PR DESCRIPTION
**Pull request by mlloydbixal**
## Jira ticket

[VOTE-951](https://cm-jira.usa.gov/browse/VOTE-951)

## Description

Fixes alignment of logos in secondary footer section, adjusts breakpoints, fixes the scaling of images, adds correct spacing to divider element.

### Pre-deploy

1. Insert any required steps to take before merging and running post-deploy.

### Post-deploy

1. Insert any required steps to take before beginning test.

### QA/Test

1. npm run build
2. Compare secondary footer to figma designs and test at different breakpoints

## Checklist for the Developer

- [ ] A link to the JIRA ticket has been included above.
- [ ] No merge conflicts exist with the target branch.
- [ ] Automated tests have passed on this PR.
- [ ] A reviewer has been designated.
- [ ] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [ ] The file changes are relevant to the task objective.
- [x] Code is readable and includes appropriate commenting.
- [x] Code standards and best practices are followed.
- [x] QA/Test steps were successfully completed, if applicable.
- [x] Applicable logs are free of errors.

### CODE REVIEW

File: web/themes/custom/votegov/src/sass/components/footer-secondary.scss [hunk 1]
```diff
@@ -3,76 +3,89 @@
...
+    }
 }
 
 .vote-social-link__icon {
+    background-color: $bg-light-cool;
+    @include u-radius-top('pill');
+    @include u-radius-bottom('pill');
     @include u-width(4);
     @include u-height(4);
+    margin-left: 18px;
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/components/footer-secondary.scss:84]:
```suggestion
    margin-left: 18px;
```

- just looking at the design there looked to be a bit to much space between the text and the logo

**Review by clmedders — COMMENTED** [review]
@mlloydbixal take a look at the code i pushed up to fix the PHP lint fail that was present and a small suggestion.. going to leave a few comments in the ticket we can talk through

File: web/themes/custom/votegov/src/sass/components/footer-secondary.scss (hunk 1 above)
Comment (mlloydbixal) [reply web/themes/custom/votegov/src/sass/components/footer-secondary.scss:84]: I thought the same, but the space on the figma file is from whitepsace at the end of the text so it doesn't actually say. I counted the pixels. I'm happy with it either way so I'll make that update. Thanks!

File: web/themes/custom/vote_gov/src/sass/custom-components/contact.scss
Comment (rayestrada) [inline web/themes/custom/vote_gov/src/sass/custom-components/contact.scss:1]: Please revert the change to this file. We are not modifying the vote_gov theme.

File: web/themes/custom/votegov/templates/component/footer--secondary-section.html.twig [hunk 2]
```diff
@@ -1,31 +1,28 @@
 {% set GSAlink = link('General Services Administration' | t, 'https://www.gsa.gov/') %}
 
 <div class="usa-footer__secondary-section" data-test="footerContact">
-  <div class="grid-container">
-    <div class="grid-row">
-      <div class="usa-footer__logo">
+  <div class="usa-footer__logo grid-container">
+      <div class="partnership-container">
```
Comment (rayestrada) [inline web/themes/custom/votegov/templates/component/footer--secondary-section.html.twig:5]: Our custom classes should use the prefix `vote-`
`vote-partnership-container`

File: web/themes/custom/votegov/src/sass/components/footer-secondary.scss [hunk 3]
```diff
@@ -3,76 +3,86 @@
...
-}
+    margin-left: 10px;
 
-.vote-social-link__icon {
-    @include u-width(4);
-    @include u-height(4);
+    img {
+        margin-left: 5px;
+        margin-top: 5px;
+        height: 22px;
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/footer-secondary.scss:86]: The X logo should be 1rem x 1rem.

**Review by rayestrada — CHANGES_REQUESTED** [review]
